.gitignore

```diff
@@ -1,4 +1,4 @@
 .DS_Store
 node_modules
-test/results
+test/results*
```

Comment:
Your test code doesn't clean up `results_css`, `results_less` and `results_scss` when test failed. And `test/results` in .gitignore doesn't match these directories. 

I think test should always be passed, but it's safer to ignore `results*` directories completely.